nx/lib/nx.ex

```diff
@@ -6329,7 +6326,7 @@ defmodule Nx do
 
     if rem(kernel_output_channels, groups) != 0 do
       raise ArgumentError,
-            "size of kernel dimension 1 must be evenly divisible by groups" <>
+            "size of kernel dimension 0 must be evenly divisible by groups" <>
```

Comment:
@seanmor5 the error said `1` but the variable `kernel_output_channels` came from `kernel_output_channels = elem(kernel_shape, 0)` so I changed the error to `0` instead of the index to `1`. Seemed safer.